### PR TITLE
updating button classes to be btn

### PR DESCRIPTION
**Pull request by katlazarova**

### CODE REVIEW

File: web/themes/custom/pineapple/components/_button.twig [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {% set button_attributes = button_attributes|default(create_attribute()).addClass([
-  'button',
-  button_type|default('button--primary'),
+  'btn',
+  button_type|default('btn--primary'),
```
Comment (ElXyfer) [inline web/themes/custom/pineapple/components/_button.twig]: I think you should update this to ```'btn--` ~ button_type|default('primary')``` then in that case button type would simply be the word either primary, secondary ect

Comment (katlazarova) [reply web/themes/custom/pineapple/components/_button.twig]: done

File: web/themes/custom/pineapple/templates/content/node--recipe--full.html.twig [hunk 2]
```diff
@@ -102,7 +103,7 @@
       </div>
     </div> <!-- End of recipe-body -->
     {% set author_node = content.field_author|field_value|first['#node'] %}
-    <aside class="recipe-sidebar">
+    <aside {{ attributes.addClass('recipe-sidebar').setAttribute('data-card', '') }}>
```
Comment (ElXyfer) [inline web/themes/custom/pineapple/templates/content/node--recipe--full.html.twig]: You don't/should not need to do this, you can just add the data attribute normally. Attributes is also a reserved variable for the node

Comment (katlazarova) [reply web/themes/custom/pineapple/templates/content/node--recipe--full.html.twig]: done

**Pull request merged**